### PR TITLE
Update property should not be read only 

### PR DESCRIPTION
**Pull request by dihan0604**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/Network/Management.Network/Generated/Models/PatchRouteFilter.cs [hunk 1]
```diff
@@ -76,10 +76,11 @@ public PatchRouteFilter(string id = default(string), IList<RouteFilterRule> rule
         public IList<RouteFilterRule> Rules { get; set; }
 
         /// <summary>
-        /// Gets a collection of references to express route circuit peerings.
+        /// Gets or sets a collection of references to express route circuit
+        /// peerings.
         /// </summary>
         [JsonProperty(PropertyName = "properties.peerings")]
-        public IList<ExpressRouteCircuitPeering> Peerings { get; private set; }
+        public IList<ExpressRouteCircuitPeering> Peerings { get; set; }
```
Comment (shahabhijeet) [inline src/SDKs/Network/Management.Network/Generated/Models/PatchRouteFilter.cs]: @dihan0604 can you provide me the swagger spec for this change?

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Network Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
-[assembly: AssemblyVersion("11.0.0.0")]
-[assembly: AssemblyFileVersion("11.0.0.0")]
+[assembly: AssemblyVersion("11.1.1.0")]
```
Comment (shahabhijeet) [inline src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs]: @dihan0604 AssemblyVersion number should change when you change the major version.
Usually you don't need to update assembly version everytime you change minor, revision number

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Comment by dihan0604** [comment]
swagger changes: https://github.com/Azure/azure-rest-api-specs/pull/1406

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
@dihan0604 this PR needs to be generated using the commit Id that you get when your swagger spec PR is merged.

**Comment by dihan0604** [comment]
swagger merged : https://github.com/Azure/azure-rest-api-specs/commit/dc20a2cdf8ae368407fe897552b7f3620972f61a

File: src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj [hunk 3]
```diff
@@ -4,6 +4,7 @@
     <PackageId>Microsoft.Azure.Management.Network</PackageId>
     <Description>Provides management capabilities for Network services.</Description>
     <AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
+    <VersionPrefix>11.0.0-preview</VersionPrefix>
```
Comment (shahabhijeet) [inline src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj]: @dihan0604 I just now noticed, why version is mentioned twice and that also separate version number?
Also as this is changing the public surface area, your major version should increase.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Comment by shahabhijeet** [comment]
@dihan0604 can you also sync with this PR, so that the versions do not conflict
https://github.com/Azure/azure-sdk-for-net/pull/3473

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**